src/hooks/useScanResults.ts

```diff
@@ -7,22 +7,12 @@ import { OR } from '../utils/tekton-results';
 import { useWorkspaceInfo } from '../utils/workspace-context-utils';
 import { useTRTaskRuns } from './useTektonResults';
 
-export const SCAN_RESULT = 'CLAIR_SCAN_RESULT';
-export const SCAN_RESULTS = 'CLAIR_SCAN_RESULTS';
-export const CVE_SCAN_RESULT = 'CVE_SCAN_RESULT';
-export const TEKTON_SCAN_RESULTS = 'TEKTON_SCAN_RESULTS';
 export const SCAN_OUTPUT = 'SCAN_OUTPUT';
 
-export const CVE_SCAN_RESULT_FIELDS = [
-  SCAN_RESULT,
-  SCAN_RESULTS,
-  CVE_SCAN_RESULT,
-  TEKTON_SCAN_RESULTS,
-  SCAN_OUTPUT,
-];
```

Comment:
We added SCAN_OUTPUT and kept old results for backward compatibility since all tasks in users workspaces are probably not updated